database/pyproject.toml

```diff
@@ -14,16 +14,16 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.10"
-core = {path = "../core", develop = true}
+core = {git = "https://github.com/calgo-lab/green-db", branch = "kvdd-update-dev-dependencies", subdirectory = "core"}
```

Comment:
This `branch = "kvdd-update-dev-dependencies` will change when merged in main, right? I assume now it's there for code/tests passing reasons